services/sidecar/src/simcore_service_sidecar/celery.py

```diff
@@ -22,20 +22,20 @@
 
 @app.task(name="comp.task", bind=True)
 def pipeline(self, user_id: str, project_id: str, node_id: str = None):
-    next_task_nodes = []
     try:
         next_task_nodes = wrap_async_call(
             run_sidecar(self.request.id, user_id, project_id, node_id)
         )
         self.update_state(state=states.SUCCESS)
+
+        if next_task_nodes:
+            for _node_id in next_task_nodes:
+                _task = app.send_task(
+                    "comp.task", args=(user_id, project_id, _node_id), kwargs={}
+                )
     except Exception:  # pylint: disable=broad-except
         self.update_state(state=states.FAILURE)
-        log.exception("Uncaught exception")
-
-    for _node_id in next_task_nodes:
-        _task = app.send_task(
-            "comp.task", args=(user_id, project_id, _node_id), kwargs={}
-        )
+        log.exception("Uncaught exception")    
```

Comment:
MINOR: could you perhaps specify a bit more about the nature of the exception :-) I am aware that the exception traceback is printed but still...